perception/sub8_pointcloud/src/OGridGen.cpp

```diff
@@ -80,14 +96,15 @@ void OGridGen::publish_ogrid(const ros::TimerEvent &)
   {
```

Comment:
This `publish_ogrid` function is getting a bit long, consider refactoring. For example, this loop could be replaced with a call to `OGridGen::project_point_cloud_to_ogrid()`